src/Clap-Core.package/ClapContext.class/README.md

```diff
@@ -0,0 +1,4 @@
+I am a class which represent a command given in the shell.
```

Comment:
Yes… but IMHO that's imprecise. Saying that this is the input stream to the parser would be better, but I'm not sure it's the best…